cpu/samd21/periph/gpio.c

```diff
@@ -41,10 +41,17 @@
  * @brief   Mapping of pins to EXTI lines, -1 means not EXTI possible
  */
 static const int8_t exti_config[2][32] = {
+#ifdef CPU_MODEL_SAMD21J18A
+    { 0,  1,  2,  3,  4,  5,  6,  7, -1,  9, 10, 11, 12, 13, 14, 15,
+      0,  1,  2,  3,  4,  5,  6,  7, 12, 13, -1, 15,  8, -1, 10, 11},
+    { 0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12, 13, 14, 15,
+      0,  1, -1, -1, -1, -1,  6,  7, -1, -1, -1, -1, -1, -1, 14, 15},
+#elif CPU_MODEL_SAMR21G18A
     {-1,  1, -1, -1,  4,  5,  6,  7, -1,  9, 10, 11, 12, 13, 14, 15,
      -1,  1,  2,  3, -1, -1,  6,  7, 12, 13, -1, 15,  8, -1, 10, 11},
     { 0, -1,  2,  3, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
-      0,  1, -1, -1, -1, -1,  6,  7, -1, -1, -1, -1,  8, -1, -1, -1},
+      0,  1, -1, -1, -1, -1,  6, 7,  -1, -1, -1, -1,  8, -1, -1, -1},
```

Comment:
Well yes :)
just to align things.
BTW, what do you think of #ifdef CPU_MODEL_SAMD21J18A and #elif CPU_MODEL_SAMR21G18A ?